pages/glossary/visual-reference-words.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Visual Reference Words
+key: visual-reference-words
+---
+
+Any word in a textnode that is included in the translated version of the following lists, where the language of the translation is the programmatically determinable language of the textnode. Some words can be translated in multible ways. In such cases each translation must be included. Some words can be spelled in multible ways. In such cases each spelling must be included.
```

Comment:
+1 @Jym77  

Plus, that lists seems to only be based on US/Europe culture/references. Maybe I'm wrong and it not.

But I personnaly do not know nothing about visual or location identification in Chinese or Vietnamese for example. Maybe they have words/expressions that have no translation at all in english.

That lists of term is good to give examples, but the rule can't limit terms to that list.

Maybe working on a little definition for every sub-visual cue (location/visual/shape/size etc...) in addition to the term lists, and not restrict to those terms.

Reply:
We tried capturing this with definitions but we where unable to do that objectively (which is necessary considering that the term is used in the applicability of the rule). 

The list is not exhaustive and might miss visual reference words from other cultures/languages and they can be added as soon as we find them. For now we are trying to catch as many applicable situations as we can with this rule.

Reply:
I think it should be made explicit that the list is not exhaustive. The current phrasing makes it look exhaustive ("any word that is included in that list" doesn't really leave room for adding words).